doc/releases/changelog-dev.md

```diff
@@ -148,6 +148,22 @@
          [ 0.162,  0.477]])
   ```
 
+* Added the possibility to compute general Renyi entropies in the `ClassicalShadow` class.
+  [(#2959)](https://github.com/PennyLaneAI/pennylane/pull/2959)
+  We can access the general Renyi-$\alpha$ entropy of a given subsystem by specifying its `wires`.
+
+  >>> shadow = ClassicalShadow(bits, recipes)
+  >>> Renyi_entropy = shadow.entropy(wires=[0, 3], alpha=1.5)
+
+  We can access the von Neumann entropy by setting `alpha=1`.
+
+  >>> shadow = ClassicalShadow(bits, recipes)
+  >>> vN_entropy = shadow.entropy(wires=[0, 3], alpha=1)
+
+  Setting `alpha=2` corresponds to the special case of computing (the logarithm of) the purity of a reduced state.
+
+  >>> purity = shadow.entropy(wires=[1, 2, 6], alpha=2)
+
```

Comment:
It would be good to have these examples show the output of the functions given the bits and recipes from a simple QNode.

Reply:
I was thinking of keeping it concise because in the part above we already have multiple examples where bits and recipes are generated

Reply:
Right, I guess we can combine all the classical shadow PRs into one section in the final release changelog, and just show all the functionality there.

Reply:
@isaacdevlugt started working on the changelog there https://github.com/PennyLaneAI/pennylane/pull/3027